bind8/config-AlmaLinux-6.0-ALL

```diff
@@ -42,15 +40,13 @@ keygen=dnssec-keygen
 signzone=dnssec-signzone
 checkconf=named-checkconf
 other_slaves=1
-restart_cmd=restart
 keygen=dnssec-keygen
 tmpl_dnssec=0
 dnssec_period=21
 tmpl_dnssec_dt=1
-dnssectools_conf=/etc/dnssec-tools/dnssec-tools.conf
```

Comment:
Why remove these lines?

Reply:
I assumed that this no longer relevant nowadays. Isn't it?

Reply:
Maybe, but let's fix one  thing at a time in this PR. I think the main issue is using the incorrect stop command?

Reply:
> but let's fix one thing at a time in this PR.

Duh, thanks for reminding me, Jamie! I'm about to get used to fixing one issue at a time!

Reply:
> I think the main issue is using the incorrect stop command?

Yes, _stop_, _start_ and _reload_ command for new distros should use `systemctl` command.